tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "lib": ["es2017","es2019.array", "DOM"],
     "declaration": true,
     "declarationMap": true,
+    "jsx": "react",
```

Comment:
optional and doesn't affect the output to my knowledge, but I get errors everywhere in vscode without it